### PR TITLE
Fix Ambiguity in workflows page and README build instructions.

### PR DESCRIPTION
**Pull request by obasekiosa**
## Motivation and Context

This change prevents ambiguity for users when using the workflows page.
It shows two separate links each for the repository and workflow file.
It also removes an obsolete re-build instruction in the README.md file.

For Issue #313: Link in the "retrieved from" column is confusing (#313) 

## Layout before change: 
image

## Layout after change:
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,10 +68,6 @@ services:
 
 See the [docker-compose.yml](docker-compose.yml) file for details.
 
-If you have modified the source code, then you may want to build the docker image locally first:
-
-    docker build -t commonworkflowlanguage/cwlviewer .
-
```
Comment (mr-c) [inline README.md:74]: Why was this removed? It worked for me..

**Review by mr-c — CHANGES_REQUESTED** [review]
Thank you @obasekiosa for this PR.

Why was an empty `docker-compose.override.yml` added?

**Comment by mr-c** [comment]
> [x] All new and existing tests passed.

The TravisCI continuous integration tests are not working right now, so there will be a delay for one of the cwlviewer maintainers to either migrate the continuous integration tests to GitHub Actions or run them locally.

File: README.md (hunk 1 above)
Comment (obasekiosa) [reply README.md:74]: > Thank you @obasekiosa for this PR.
> 
> Why was an empty `docker-compose.override.yml` added?

The empty file was a mistake.



> Why was this removed? It worked for me.

It seems to be doing the exact same thing as the override file and for some reason, it didn't work for me.
Changes I made did not show up until I used the override file.
Also, the instruction seems to be a duplicate of the one above it. What to do after a change has been made to the source code.
